weblate_web/templates/user.html

```diff
@@ -23,7 +23,7 @@ <h1>{% trans "Your services" %}</h1>
                         {{ service.get_status_display }}
                         {% if not service.check_in_limits %}
                         <div class="alert error">
-                        {% trans "Exceeding service limits" %}
+                        {% trans "Exceeds selected service period" %}
```

Comment:
It's actually about limits - number strings/languages (as listed in https://weblate.org/hosting/)

Reply:
In my head "limitations" makes more sense, will think about it.